thirdeye/thirdeye-frontend/app/pods/components/rootcause-select-comparison-range/component.js

```diff
@@ -40,58 +37,64 @@ export default Component.extend({
   showBaselineModal: false,
   showForecastTimeRanges: false,
   customBaselineValue: 'wo1w',
-
+  // eslint-disable-next-line ember/avoid-leaking-state-in-ember-objects
```

Comment:
This entire object can be defined outside the component as a const and referenced here so as to avoid bypassing eslint.